recipes/freecad/meta.yaml

```diff
@@ -0,0 +1,99 @@
+{% set name = "freecad" %}
+{% set version = "0.18b0" %}
+{% set sha256 = "1462df34caa67791bb7512f6c464f16a15d49f0d9f777996cb4e7b8174c57191" %}
+{% set commit = "e49a5af3ae2f976017f1156bdc113594f81d3fb5" %}
+
+
+package:
+    name: {{ name }}
+    version: {{ version }}
+
+source:
+    fn: {{ name }}_{{ version }}.tar.gz
+    url: https://github.com/FreeCAD/FreeCAD/archive/{{ commit }}.tar.gz
+    sha256: {{ sha256 }}
+    patches:
+        - osx.patch
+        - smesh.patch
+        - win_unicode.patch
+
+build:
+    number: 0
+    skip: True  # [py < 36]
+
+requirements:
+    build:
+        - {{ compiler("cxx") }}
+        - ninja
+        - cmake
+    host:
+        - nomkl  # [unix]
```

Comment:
This shouldn't be used. If you would like to force the `blas` from `conda-forge`, it should be possible to do the following `blas 1.1 openblas`.

xref: https://github.com/conda-forge/numpy-feedstock/blob/2b63ffa6527587aa5b06bb80178c088ed1056b51/recipe/meta.yaml#L16